### PR TITLE
feat(starlark): improve assertion parsing

### PR DESCRIPTION
**Pull request by amaanq**
Title

### CODE REVIEW

File: queries/starlark/highlights.scm [hunk 1]
```diff
@@ -256,6 +255,11 @@
 
 ;; Starlark-specific
 
+;; Assertion calls
+(assert_keyword) @keyword
+
+(assert_builtin) @function.builtin
```
Comment (ObserverOfTime) [inline queries/starlark/highlights.scm:261]: What's the difference between these two?
Can you use `@exception` for either or both?

Comment (amaanq) [reply queries/starlark/highlights.scm:261]:
```
assert a = b
assert_eq(a = b)
assert.eq(a = b)
```

Are assert functions normally captured under `@exception`?

Comment (ObserverOfTime) [reply queries/starlark/highlights.scm:261]: Probably not the functions, but the first `assert` should be `@exception`.

Comment (amaanq) [reply queries/starlark/highlights.scm:261]: Python has it as `@keyword`

Comment (amaanq) [reply queries/starlark/highlights.scm:261]: Python has it as `@keyword`

Comment (ObserverOfTime) [reply queries/starlark/highlights.scm:261]: We might be using it inconsistently.
Let's keep it consistent with python here.

**Review by ObserverOfTime — APPROVED** [review]

File: queries/starlark/highlights.scm (hunk 1 above)
Comment (amaanq) [reply queries/starlark/highlights.scm:261]: Agreed, especially since Starlark is just a dialect of Python

**Pull request merged**